src/components/FilterPanel/FilterAccordionItem/_mixins.scss

```diff
@@ -34,32 +39,37 @@
     &__heading {
       /// Scroller padding.
       /// @type Length
-      $filter-subcategory__accordion__spacing__padding: $carbon--spacing-02;
+      $filter-accordion-item__content__spacing__padding: $carbon--spacing-02;
 
-      padding-top: $filter-subcategory__accordion__spacing__padding;
-      padding-bottom: $filter-subcategory__accordion__spacing__padding;
+      padding-top: $filter-accordion-item__content__spacing__padding;
+      padding-bottom: $filter-accordion-item__content__spacing__padding;
       color: $text-02;
     }
 
     &__content {
-      padding: 0 0 0 $filter-subcategory__spacing__padding;
+      padding: 0 0 0 $filter-accordion-item__spacing__padding;
     }
   }
 
+  // Reduce the width of the label so long labels with counts overflow out of the panel.
+  #{$carbon--accordion}__title {
+    width: calc(100% - 2.8em);
```

Comment:
Yeah, I meant to use `em`. The value comes from the space that the arrow next to the accordion item title takes. Unfortunately, this is the only way I have found to prevent this from happening: 

![image](https://user-images.githubusercontent.com/5973294/72567144-e6595080-387a-11ea-9a02-1796c7e378bc.png)